src/ILCompiler.Compiler/src/ILCompiler.Compiler.csproj

```diff
@@ -255,6 +255,15 @@
     <Compile Include="..\..\Common\src\TypeSystem\IL\Stubs\PInvokeILEmitter.cs">
       <Link>IL\Stubs\PInvokeILEmitter.cs</Link>
     </Compile>
+    <Compile Include="..\..\Common\src\TypeSystem\Interop\IL\Marshaller.cs">
+      <Link>IL\Interop\Marshaller.cs</Link>
```

Comment:
Hmm ... all PInvoke declarations are linked under `Interop\...` ... maybe these links should be `TypeSystem\Interop\IL` or something similar to differentiate from the PInvoke declarations.